src/providers/common.ts

```diff
@@ -116,15 +116,20 @@ export class Os {
   /**
   * Linux
   *
-  * @deprecated use {@link LINUX_UBUNTU} or {@link LINUX_AMAZON_2} or {@link LINUX_AMAZON_2023}
+  * @deprecated use {@link LINUX_UBUNTU}, {@link LINUX_UBUNTU_2404}, {@link LINUX_AMAZON_2} or {@link LINUX_AMAZON_2023}
   */
   public static readonly LINUX = Os.of('Linux');
 
   /**
-   * Ubuntu Linux
+   * Ubuntu Linux 22.04
    */
   public static readonly LINUX_UBUNTU = Os.of('Ubuntu Linux');
 
+  /**
```

Comment:
We should also add `LINUX_UBUNTU_2204 = Os.of('Ubuntu Linux')` for completeness and clarity.